dora/core/client/fs/src/test/java/alluxio/client/file/cache/LocalCacheManagerTest.java

```diff
@@ -690,10 +694,14 @@ public void syncRestoreUnknownFile() throws Exception {
     mPageMetaStore = new DefaultPageMetaStore(ImmutableList.of(dir));
     mCacheManager = LocalCacheManager.create(mCacheManagerOptions, mPageMetaStore);
     assertEquals(CacheManager.State.READ_WRITE, mCacheManager.state());
-    assertEquals(0, mCacheManager.get(PAGE_ID1, PAGE1.length, mBuf, 0));
-    assertEquals(0, mCacheManager.get(pageUuid, PAGE2.length, mBuf, 0));
+    assertEquals(PAGE1.length, mCacheManager.get(PAGE_ID1, PAGE1.length, mBuf, 0));
+    assertEquals(PAGE2.length, mCacheManager.get(pageUuid, PAGE2.length, mBuf, 0));
```

Comment:
if invalid pages are deleted, why should cache manager be able to read anything out of these invalid pages?

Reply:
These are two valid page files, inserted in previous lines:
```
    dir.getPageStore().put(PAGE_ID1, PAGE1);
    dir.getPageStore().put(pageUuid, PAGE2);
```

The invalidate page file is created by manually creating a file in UFS:
```
FileUtils.createFile(Paths.get(rootDir, "invalidPageFile").toString());
```
This invalid page file will be unrecognized and will be deleted it.

Reply:
Please refer to LocalCacheManagerIntegrationTest::loadCacheWithInvalidPageFile()

I didn't add checking to verify the invalid page file is deleted.
The checking and verification is done in LocalCacheManagerIntegrationTest::loadCacheWithInvalidPageFile()

Reply:
I figured this out. `PAGE_ID1` and `pageUuid` are both valid pages. In the original implementation they were deleted because `invalidPageFile` existed and the logic was to clear the entire cache directory if a single invalid page was found.
After this PR, only invalid pages will be deleted and valid ones are preserved.